USAGE_DATA.md

```diff
@@ -0,0 +1,15 @@
+# Telemetry data collection
+
+Telemetry events can be collected by the LSP client program.
+
+The following information is emitted when the language server starts:
+
+ * JVM information:
+    * Whether it is being run with Java or as a GraalVM native image (binary)
```

Comment:
in our case, currently always return false but it allows to have teh same structural metric data than xml ls